lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h

```diff
@@ -668,7 +668,8 @@ template <> inline void SPIRVMap<Capability, std::string>::init() {
       "CooperativeMatrixInvocationInstructionsINTEL");
   add(internal::CapabilityCooperativeMatrixCheckedInstructionsINTEL,
       "CooperativeMatrixCheckedInstructionsINTEL");
-  add(internal::CapabilityRegisterLimitsINTEL, "RegisterLimitsINTEL");
+  add(internal::CapabilityTaskSequenceINTEL, "TaskSequenceINTEL");
```

Comment:
This seems to cause test fails. I suppose you would have noticed as well. Will review after fixing. Thanks much.

Reply:
Thanks!